testsuite/run_sets/build_validation/build_validation_retail_sle12.yml

```diff
@@ -5,9 +5,5 @@
 ## Prepare configuration for terminal
 - features/build_validation/retail/sle12sp5_terminal_prepare.feature
```

Comment:
Shouldn't it be before `features/build_validation/retail/sle12sp5_buildhost_build_kiwi_image.feature`?